2012/07/06 02:05:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:05:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:05:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:05:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:05:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:05:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:05:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:08:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:08:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:08:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:49 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:49 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:49 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'Wibi'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:11:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:12:04 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Invalid object name 'HumanResources.Employee'.. The SQL statement executed was: SELECT * FROM HumanResources.Employee.
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (181)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/06 02:12:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Invalid object name 'HumanResources.Employee'.. The SQL statement executed was: SELECT * FROM HumanResources.Employee' in C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php(390): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(181): CDbCommand->queryAll()
#2 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#9 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:12:16 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[08004]: [Microsoft][SQL Server Native Client 11.0][SQL Server]The server principal "NT AUTHORITY\SYSTEM" is not able to access the database "AdventureWorks" under the current security context.. The SQL statement executed was: USE AdventureWorks; SELECT * FROM HumanResources.Employee.
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (181)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/06 02:12:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[08004]: [Microsoft][SQL Server Native Client 11.0][SQL Server]The server principal "NT AUTHORITY\SYSTEM" is not able to access the database "AdventureWorks" under the current security context.. The SQL statement executed was: USE AdventureWorks; SELECT * FROM HumanResources.Employee' in C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php(390): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(181): CDbCommand->queryAll()
#2 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#9 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:12:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:13:01 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:13:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:15:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:27:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:37:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:39:57 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:42:08 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:42:09 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 02:53:56 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 03:00:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 03:00:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 03:00:48 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 03:00:59 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 03:03:25 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 03:03:25 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 04:26:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(178): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 04:29:40 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Could not find stored procedure 'sphelp'.. The SQL statement executed was: EXEC sphelp.
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (181)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/06 04:29:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Could not find stored procedure 'sphelp'.. The SQL statement executed was: EXEC sphelp' in C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php(390): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(181): CDbCommand->queryAll()
#2 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#9 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 04:29:44 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Incorrect syntax near '-'.. The SQL statement executed was: EXEC -sphelp.
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (181)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/06 04:29:44 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Incorrect syntax near '-'.. The SQL statement executed was: EXEC -sphelp' in C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php(390): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(181): CDbCommand->queryAll()
#2 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#9 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 04:29:54 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Could not find stored procedure 'sphelptext'.. The SQL statement executed was: EXEC sphelptext.
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (181)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/06 04:29:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Could not find stored procedure 'sphelptext'.. The SQL statement executed was: EXEC sphelptext' in C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php(390): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(181): CDbCommand->queryAll()
#2 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#9 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 05:08:58 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\portal\protected\extensions\phpexcel\php-excel.class.php:168) (C:\xampp\htdocs\portal\protected\extensions\phpexcel\php-excel.class.php:160)
Stack trace:
#0 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(197): SiteController->generateExcel()
#1 C:\xampp\htdocs\portal\protected\views\site\result.php(2): SiteController->actionDownload()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(127): require()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(96): SiteController->renderInternal()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(870): SiteController->renderFile()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(783): SiteController->renderPartial()
#6 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(189): SiteController->render()
#7 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): SiteController->runAction()
#10 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#11 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): SiteController->runActionWithFilters()
#12 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): SiteController->run()
#13 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#14 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#15 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=site/test
in C:\xampp\htdocs\portal\protected\extensions\phpexcel\php-excel.class.php (160)
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (229)
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (197)
2012/07/06 05:20:13 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'No query found.' in C:\xampp\htdocs\portal\protected\controllers\SiteController.php:164
Stack trace:
#0 [internal function]: SiteController->actionTest('', '')
#1 C:\xampp\htdocs\Yii_portal\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs(Object(SiteController), Array)
#2 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(48): CAction->runWithParamsInternal(Object(SiteController), Object(ReflectionMethod), Array)
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#9 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 05:20:42 [error] [php] Missing argument 1 for SiteController::generateText(), called in C:\xampp\htdocs\portal\protected\controllers\SiteController.php on line 167 and defined (C:\xampp\htdocs\portal\protected\controllers\SiteController.php:238)
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\web\actions\CAction.php(107): ReflectionMethod->invokeArgs()
#1 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(48): CInlineAction->runWithParamsInternal()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): SiteController->runAction()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): SiteController->runActionWithFilters()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): SiteController->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#8 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#9 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=site/test&download=txt
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (238)
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (167)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/06 08:24:22 [error] [php] include(CGridView.php) [<a href='function.include'>function.include</a>]: failed to open stream: No such file or directory (C:\xampp\htdocs\Yii_portal\framework\YiiBase.php:418)
Stack trace:
#0 C:\xampp\htdocs\portal\protected\components\FlowGridView.php(3): spl_autoload_call()
#1 C:\xampp\htdocs\Yii_portal\framework\YiiBase.php(316): require()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CWidgetFactory.php(147): import()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(147): CWidgetFactory->createWidget()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(173): SiteController->createWidget()
#5 C:\xampp\htdocs\portal\protected\views\site\result.php(22): SiteController->widget()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(127): require()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(96): SiteController->renderInternal()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(870): SiteController->renderFile()
#9 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(783): SiteController->renderPartial()
#10 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(186): SiteController->render()
#11 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#12 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#13 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): SiteController->runAction()
#14 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#15 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): SiteController->runActionWithFilters()
#16 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): SiteController->run()
#17 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#18 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#19 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=site/test&download=txt
in C:\xampp\htdocs\portal\protected\components\FlowGridView.php (3)
in C:\xampp\htdocs\portal\protected\views\site\result.php (22)
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (186)
2012/07/06 09:06:19 [error] [php] Undefined variable: body (C:\xampp\htdocs\portal\protected\components\FlowGridView.php:16)
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\zii\widgets\CBaseListView.php(141): preg_replace_callback()
#1 C:\xampp\htdocs\Yii_portal\framework\zii\widgets\CBaseListView.php(126): FlowGridView->renderContent()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(174): FlowGridView->run()
#3 C:\xampp\htdocs\portal\protected\views\site\result.php(19): SiteController->widget()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(127): require()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CBaseController.php(96): SiteController->renderInternal()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(870): SiteController->renderFile()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(783): SiteController->renderPartial()
#8 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(186): SiteController->render()
#9 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#10 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#11 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): SiteController->runAction()
#12 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#13 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): SiteController->runActionWithFilters()
#14 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): SiteController->run()
#15 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#16 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#17 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=site/test
in C:\xampp\htdocs\portal\protected\components\FlowGridView.php (16)
in C:\xampp\htdocs\portal\protected\views\site\result.php (19)
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (186)
2012/07/06 12:15:24 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Incorrect syntax near the keyword 'FROM'.. The SQL statement executed was: SELECT TOP 5 FROM Production.Product.
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (180)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/06 12:15:24 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Incorrect syntax near the keyword 'FROM'.. The SQL statement executed was: SELECT TOP 5 FROM Production.Product' in C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php:528
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbCommand.php(390): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(180): CDbCommand->queryAll()
#2 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#9 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#10 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 14:44:53 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08001]: [Microsoft][SQL Server Native Client 11.0]Named Pipes Provider: Could not open a connection to SQL Server [2]. ' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 14:47:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08001]: [Microsoft][SQL Server Native Client 11.0]Named Pipes Provider: Could not open a connection to SQL Server [2]. ' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 14:47:56 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08001]: [Microsoft][SQL Server Native Client 11.0]Named Pipes Provider: Could not open a connection to SQL Server [2]. ' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/06 14:54:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/07 01:37:32 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08001]: [Microsoft][SQL Server Native Client 11.0]Named Pipes Provider: Could not open a connection to SQL Server [2]. ' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/07 01:59:29 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08001]: [Microsoft][SQL Server Native Client 11.0]Named Pipes Provider: Could not open a connection to SQL Server [2]. ' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/09 02:07:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "tex".' in C:\xampp\htdocs\Yii_portal\framework\web\CController.php:484
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(271): CController->missingAction('tex')
#1 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('tex')
#2 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/tex')
#3 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#4 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/portal/index.php?r=site/tex
---
2012/07/09 02:07:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[08001]: [Microsoft][SQL Server Native Client 11.0]Named Pipes Provider: Could not open a connection to SQL Server [2]. ' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/09 02:08:12 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/09 02:09:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/09 02:20:57 [error] [php] Undefined variable: dsn (C:\xampp\htdocs\portal\protected\controllers\SiteController.php:176)
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): SiteController->runAction()
#1 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): SiteController->runActionWithFilters()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): SiteController->run()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#5 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#6 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=site/test
in C:\xampp\htdocs\portal\protected\controllers\SiteController.php (176)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/09 03:46:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000]: [Microsoft][SQL Server Native Client 11.0][SQL Server]Login failed for user 'NT AUTHORITY\SYSTEM'.' in C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php:382
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\db\CDbConnection.php(331): CDbConnection->open()
#1 C:\xampp\htdocs\Yii_portal\framework\base\CComponent.php(153): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\portal\protected\controllers\SiteController.php(177): CComponent->__set('active', true)
#3 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): SiteController->actionTest()
#4 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): CController->run('test')
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController('site/test')
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/portal/index.php?r=site/test
---
2012/07/09 06:14:41 [error] [php] Missing argument 2 for QueryController::checkStatements(), called in C:\xampp\htdocs\portal\protected\controllers\QueryController.php on line 61 and defined (C:\xampp\htdocs\portal\protected\controllers\QueryController.php:254)
Stack trace:
#0 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): QueryController->runAction()
#2 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilter.php(41): CFilterChain->run()
#3 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(1146): CAccessControlFilter->filter()
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CInlineFilter.php(59): QueryController->filterAccessControl()
#5 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#6 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#7 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): QueryController->runActionWithFilters()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): QueryController->run()
#9 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#10 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#11 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=query/add
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (254)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (61)
in C:\xampp\htdocs\portal\index.php (13)
2012/07/09 06:15:54 [error] [php] Undefined offset: 1 (C:\xampp\htdocs\portal\protected\controllers\QueryController.php:298)
Stack trace:
#0 C:\xampp\htdocs\portal\protected\controllers\QueryController.php(61): QueryController->checkStatements()
#1 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): QueryController->actionAdd()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): QueryController->runAction()
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(1146): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\Yii_portal\framework\web\filters\CInlineFilter.php(59): QueryController->filterAccessControl()
#7 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): QueryController->runActionWithFilters()
#10 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): QueryController->run()
#11 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#12 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=query/add
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (298)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (277)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (257)
2012/07/09 06:16:34 [error] [php] Undefined offset: 1 (C:\xampp\htdocs\portal\protected\controllers\QueryController.php:298)
Stack trace:
#0 C:\xampp\htdocs\portal\protected\controllers\QueryController.php(61): QueryController->checkStatements()
#1 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): QueryController->actionAdd()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): QueryController->runAction()
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(1146): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\Yii_portal\framework\web\filters\CInlineFilter.php(59): QueryController->filterAccessControl()
#7 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): QueryController->runActionWithFilters()
#10 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): QueryController->run()
#11 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#12 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=query/add
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (298)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (277)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (257)
2012/07/09 06:17:19 [error] [php] Undefined offset: 1 (C:\xampp\htdocs\portal\protected\controllers\QueryController.php:298)
Stack trace:
#0 C:\xampp\htdocs\portal\protected\controllers\QueryController.php(61): QueryController->checkStatements()
#1 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): QueryController->actionAdd()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): QueryController->runAction()
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(1146): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\Yii_portal\framework\web\filters\CInlineFilter.php(59): QueryController->filterAccessControl()
#7 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): QueryController->runActionWithFilters()
#10 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): QueryController->run()
#11 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#12 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=query/add
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (298)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (277)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (257)
2012/07/09 06:17:28 [error] [php] Undefined offset: 1 (C:\xampp\htdocs\portal\protected\controllers\QueryController.php:298)
Stack trace:
#0 C:\xampp\htdocs\portal\protected\controllers\QueryController.php(61): QueryController->checkStatements()
#1 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): QueryController->actionAdd()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): QueryController->runAction()
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(1146): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\Yii_portal\framework\web\filters\CInlineFilter.php(59): QueryController->filterAccessControl()
#7 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): QueryController->runActionWithFilters()
#10 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): QueryController->run()
#11 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#12 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=query/add
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (298)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (277)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (257)
2012/07/09 06:26:54 [error] [php] Undefined offset: 0 (C:\xampp\htdocs\portal\protected\controllers\QueryController.php:301)
Stack trace:
#0 C:\xampp\htdocs\portal\protected\controllers\QueryController.php(61): QueryController->checkStatements()
#1 C:\xampp\htdocs\Yii_portal\framework\web\actions\CInlineAction.php(50): QueryController->actionAdd()
#2 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(309): CInlineAction->runWithParams()
#3 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(134): QueryController->runAction()
#4 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilter.php(41): CFilterChain->run()
#5 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(1146): CAccessControlFilter->filter()
#6 C:\xampp\htdocs\Yii_portal\framework\web\filters\CInlineFilter.php(59): QueryController->filterAccessControl()
#7 C:\xampp\htdocs\Yii_portal\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#8 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(292): CFilterChain->run()
#9 C:\xampp\htdocs\Yii_portal\framework\web\CController.php(266): QueryController->runActionWithFilters()
#10 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(276): QueryController->run()
#11 C:\xampp\htdocs\Yii_portal\framework\web\CWebApplication.php(135): CWebApplication->runController()
#12 C:\xampp\htdocs\Yii_portal\framework\base\CApplication.php(162): CWebApplication->processRequest()
#13 C:\xampp\htdocs\portal\index.php(13): CWebApplication->run()
REQUEST_URI=/portal/index.php?r=query/add
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (301)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (277)
in C:\xampp\htdocs\portal\protected\controllers\QueryController.php (257)
